src/docs/sdk/craft-quick-start.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: "craft Quick Start"
+---
+
+this guide goes over the minimal setup needed to publish something via
+[publish] and [craft].  this example will use a pypi package.
+
+[publish]: https://github.com/getsentry/publish
+[craft]: https://github.com/getsentry/craft
+
+## the first release
+
+this guide assumes this thing has never been released before and has no
+existing tags.  if that's not the case feel free to skip this section!
+
+a tag must exist for the "beginning" of the repository due to [a limitation] --
+it is recommended to tag the first commit of the repository before any
+meaningful history such that the first changelog includes useful context.
+
+```bash
+git tag 0.0.0 "$(git log -1 --reverse --format=%h)"
+git push origin --tags
+```
+
+[a limitation]: https://github.com/getsentry/craft/issues/342
+
+## ci requirements
+
+first we need to ensure your repository responds to `release/**` branches
+and produces an artifact with the right name
+
+for our example repository that's done using [push: branches] and an
```

Comment:
```suggestion
for our example repository that's done by ensuring the following in a GitHub action:
1. [push: branches]
    ```yaml
    on:
      pull_request:
      push:
        branches: [main, test-me-*, release/**]
    ```
```

Reply:
there's other ways to do this so I didn't want to prescribe the exact solution